File: bidly-backend/src/main/java/com/bidly/bidly_backend/model/Notificacion.java
package com.bidly.bidly_backend.model;

import jakarta.persistence.*;
import lombok.*;
import java.time.LocalDateTime;

@Entity
@Table(name = "notificaciones")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Notificacion {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long identificador;

    @ManyToOne
    @JoinColumn(name = "cliente")
    private Cliente cliente;

    private String tipo;
    private String mensaje;
    private String leida;

    @Column(name = "fechahora")
    private LocalDateTime fechaHora;
}

File: bidly-fastapi/app/models/notificacion.py
from sqlalchemy import Column, Integer, String, DateTime, ForeignKey
from app.database import Base


class Notificacion(Base):
    __tablename__ = "notificaciones"

    identificador = Column(Integer, primary_key=True, autoincrement=True)
    cliente       = Column(Integer, ForeignKey("clientes.identificador"))
    tipo          = Column(String)
    mensaje       = Column(String)
    leida         = Column(String, default="no")
    fechahora     = Column(DateTime)
